### ISSUE TITLE
Symbols in Bash on Ubuntu on Windows don't work

### ISSUE THREAD

**Issue by davidhatten** [aurelien-rainone/tmux-gitbar#49]
Hi, I'm using this in Bash on Ubuntu on Windows. The arrow symbol is showing up, but nothing else is.

In Courier:
![image](https://user-images.githubusercontent.com/11279749/31046362-1a175386-a5c5-11e7-8d4b-7e0adc36cb31.png)

In Consolas
![image](https://user-images.githubusercontent.com/11279749/31046366-280727dc-a5c5-11e7-9812-9be7bdf9c6ac.png)

**Comment by arl** [comment]
Hi David,

thanks for reporting. Actually I never tried *tmux-gitbar* on Windows.
I can see 2 ways to solve your problem:
 

1. Find a font in which the missing symbols are defined. I point you to the [Font section of tmux-gibar documentation](https://github.com/aurelien-rainone/tmux-gitbar#font)

2. You can easily replace the problematic symbols with other symbols, for that check out the [Symbols section of the doc](https://github.com/aurelien-rainone/tmux-gitbar#symbols). All default symbols can be redefined in your tmux-gitbar configuration file, find the complete list in [`./lib/tmux-gitbar.sh`](https://github.com/aurelien-rainone/tmux-gitbar/blob/053f76971a5d3482782e29903f503b1fb6667a49/lib/tmux-gitbar.sh#L19)

**Comment by davidhatten** [comment]
I'll see about getting Consolas Powerline installed, if that's applicable in my case.

Are you familiar with the Linux Subsystem/Bash on Ubuntu on Windows framework? It's not entirely correct to say that this is running on Windows, but nor is it entirely wrong to say that. It may exist on Windows, but it is 100% a bash terminal that reads from `~/.bashrc` at runtime. It's some crazy magic that's probably contributing to whatever issue is happening here. I'll dig around.

**Comment by davidhatten** [comment]
Using the referenced Consolas Powerline in the Fonts section results in the same artifacts as Consolas.

It seems like this is happening with every font. There's probably something deeper going on, and the issue is likely with Bash on Ubuntu on Windows and not with your library.

**Comment by arl** [comment]
Sorry to hear that. Maybe you could go with replacing the characters in
tmux gitbar config file

I am interested in knowing how you resolve the font problem. If you find a
suitable font for your system I'd include it in the doc as I guess Ubuntu
subsystem for Window will be more and more used in the future.

Cheers

**Comment by arl** [comment]
Hey @davidhatten
Did you find a solution/workaround?

**Comment by aljavier** [comment]
I was in the same situation. What I did was that I first installed [consoles-font-powerline](https://github.com/runsisi/consolas-font-for-powerline), which as @davidhatten said did not fix the issue.
Then I went to the `.tmux-gitbar/lib/tmux-gitbar.sh` directory and verified there that the characters were unreadable. So I decided to copy those characters variables on my `.tmux-gitbar.conf` and replace the characters as needed. For some reason, even having the fonts installed and/or copy-pasting the characters from the file here on the repository (where the fonts are readable correctly) it does not display well on Windows.

So, open the `Character Map` app on Windows, and select `Powerline Consolas`, search over there whatever character you want to use, I did not find some of the ones I see here in the default file but I did not search that much I'm in a hurry. Select the character and then go to your  `.tmux-gitbar.conf` and change it there. This is part of my config now:

![image](https://user-images.githubusercontent.com/1992392/41299027-8996be16-6e30-11e8-9a18-75b7455a1d76.png)

```
BRANCH_SYMBOL=" "
STAGED_SYMBOL="●"
CONFLICT_SYMBOL="х"
CHANGED_SYMBOL="[+]"
UNTRACKED_SYMBOL="..."
STASHED_SYMBOL=""
CLEAN_SYMBOL="√"
AHEAD_SYMBOL="↑·"
BEHIND_SYMBOL="↓·"
PREHASH_SYMBOL=":"
```
![image](https://user-images.githubusercontent.com/1992392/41351366-21fc744e-6ee4-11e8-93b3-3c44930ea847.png)


And here is the Character Map app in action:
![image](https://user-images.githubusercontent.com/1992392/41299070-a3385258-6e30-11e8-9ceb-3c07d76b36f7.png)

Regards,

A.J.
